src/constants/actions.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // action types
+  TRANSITION: 'ACTION_TYPE/TRANSITION',
+  METADATA: 'ACTION_TYPE/METADATA',
+  PROGRESS: 'ACTION_TYPE/PROGRESS',
+  // transition actions
+  REJECT: 'TRANSITION/REJECT',
```

Comment:
a bit confused now. So, in the end, will you have just routing `transition`, `metadata`, and `progress`? 

Or you will have all of these as url routings? (`transition/reject`, `transition/verify`, ...`progress/delete_progress`)